src/unix/sunos.c

```diff
@@ -542,12 +547,14 @@ int uv_fs_event_start(uv_fs_event_t* handle,
 }
 
 
-int uv_fs_event_stop(uv_fs_event_t* handle) {
+static int i_uv_fs_event_stop(uv_fs_event_t* handle) {
```

Comment:
style nit: internal functions are identified with `uv__` prefix, not `i_`
```suggestion
static int uv__fs_event_stop(uv_fs_event_t* handle) {
```

Reply:
Thanks, changed.